lib/web_ui/lib/src/engine/html/picture.dart

```diff
@@ -270,9 +269,9 @@ class PersistedPicture extends PersistedLeafSurface {
           _predictTrend(bottomwardDelta, _exactLocalCullRect!.height),
     ).intersect(localPaintBounds!);
 
-    final bool localCullRectChanged = _optimalLocalCullRect != newLocalCullRect;
+    _requiresRepaint = _optimalLocalCullRect != newLocalCullRect;
     _optimalLocalCullRect = newLocalCullRect;
-    return localCullRectChanged;
+    return;
```

Comment:
this is the last line in the function. No `return` necessary.

Reply:
Done.